decidim-process-extended/app/services/decorators/decidim/participatory_processes/participatory_process_search_decorator.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require_dependency 'decidim/participatory_processes/participatory_process_search'
+
+Decidim::ParticipatoryProcesses::ParticipatoryProcessSearch.class_eval do
+
+  def initialize(options = {})
+    super(Decidim::ParticipatoryProcess.all, options).where(decidim_participatory_process_group_id: Rails.application.config.process)
```

Comment:
So cool!

Reply:
Oliver, what should we do here? Leave it as it is currently or change to this?
```suggestion
    super(Decidim::ParticipatoryProcess.all.where(decidim_participatory_process_group_id: Rails.application.config.process), options)
```